aws-sso-instanceaccesscontrolattributeconfiguration/docs/README.md

```diff
@@ -47,20 +45,9 @@ _Maximum_: <code>1224</code>
 _Pattern_: <code>arn:aws:sso:::instance/(sso)?ins-[a-zA-Z0-9-.]{16}</code>
 
 _Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)
-
-#### InstanceAccessControlAttributeConfiguration
```

Comment:
I don't think this page links to [instanceaccesscontrolattributeconfiguration.md](instanceaccesscontrolattributeconfiguration.md) anymore, which could confuse people. I think the deprecation notice should be on this page (linking to the other page for the schema for `InstanceAccessControlAttributeConfiguration`)